plutus-tx-plugin/test/TH/Spec.hs

```diff
@@ -12,7 +12,7 @@
 {-# OPTIONS_GHC -Wno-orphans #-}
 {-# OPTIONS_GHC   -g #-}
 
-module TH.Spec (tests) where
+module TH.Spec (goldenEvalCekLog, tests) where
```

Comment:
I can have another commit porting `goldenEvalCekLog` to `TestLib`?

Reply:
Yeah, why not.

Reply:
I have moved those functions to `plutus-tx-plugin/test/Lib.hs`